client/signup/steps/reader-landing/content.jsx

```diff
@@ -1,5 +1,5 @@
 /** @format */
-
+/* eslint-disable */
```

Comment:
@torres126 which lint rule are we trying to ignore here?

Reply:
No clue what that was for to be honest, clearly not a necessary one though. Removed. :)